applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py

```diff
@@ -315,6 +315,19 @@ def AddVariables(self):
         KratosMultiphysics.Logger.PrintInfo(self.__class__.__name__, "Fluid solver variables added correctly.")
 
     def Initialize(self):
+    	# My shock process
+        settings = KratosMultiphysics.Parameters("""{
+            "model_part_name" : "FluidModelPart",
+            "calculate_nodal_area_at_each_step" : false,
+            "shock_sensor" : true,
+            "shear_sensor" : true,
+            "thermal_sensor" : false,
+            "thermally_coupled_formulation" : false
+        }""")
+
+        self.shock_process = KratosCFD.ShockCapturingProcess(self.model, settings)
+        self.shock_process.Check()
+        self.shock_process.ExecuteInitialize()
```

Comment:
this should be added through the project parameters and not directly in the solver

Reply:
Done. We will add it to a sandbox version for the project.